assets/js/highlight.js

```diff
@@ -1,8 +1,8 @@
-import hljs from 'highlight.js/lib/core';
+import hljs from 'highlight.js';
```

Comment:
this would load all languages, making the build a lot bigger. Why doing that change ?

Reply:
@stof, I don't know why I did this :( . Fixed.